
const isLogin = async(req,res,next)=>{

    try {
        


    } catch (error) {
        console.log(error.message);
    }

}



const isLo = async(req,res,next)=>{

    try {
        
        

    } catch (error) {
        console.log(error.message);
    }


}